### PR TITLE
DDCE-2707 - Migrate to HMRC-Mongo

### PR DESCRIPTION
**Pull request by frikit**

### CODE REVIEW

File: app/repositories/SessionRepository.scala [hunk 1]
```diff
@@ -16,66 +16,54 @@
...
+
+  private def byId(id: String) = Filters.eq("_id", id)
+
+  override def get(id: String): Future[Option[UserAnswers]] = {
+    val modifier = Updates.set("lastUpdated", LocalDateTime.now)
+
+    val options = new FindOneAndUpdateOptions()
+      .upsert(false)
+      .returnDocument(ReturnDocument.AFTER)
+    collection.findOneAndUpdate(byId(id), modifier, options).headOption()
```
Comment (DannyPennington97) [inline app/repositories/SessionRepository.scala:57]: Is there a reason for updating the "lastUpdated" value in this method? Previously it was a read only

**Review by DannyPennington97 — COMMENTED** [review]
One question, all else looking good and tests all passing

File: app/repositories/SessionRepository.scala (hunk 1 above)
Comment (frikit) [reply app/repositories/SessionRepository.scala:57]: If user start to answer questions and dont finish in 15 mins timeframe then they wont be able to submit even if they submited one question per 3 mins for example just as slow pace, in this case we will update usernaswers everytime system request it and then user will have 15 mins timeout only when they are really away for 15 mins, hope that makes sense the same logic is in other services

Comment (DannyPennington97) [reply app/repositories/SessionRepository.scala:57]: Cool yeah that makes sense, ready to merge then!

**Review by DannyPennington97 — APPROVED** [review]

**Pull request merged**